usaspending_api/database_scripts/matviews/drop_old_matviews.sql

```diff
@@ -1,4 +1,4 @@
 DROP MATERIALIZED VIEW IF EXISTS mv_agency_autocomplete_old;
 DROP MATERIALIZED VIEW IF EXISTS mv_covid_financial_account_old;
-DROP MATERIALIZED VIEW IF EXISTS subaward_view_old;
+DROP MATERIALIZED VIEW IF EXISTS summary_state_view_old;
```

Comment:
Why are the `subaward` fields being replaced with `summary_state_view` in these three files instead of being removed altogether?

Reply:
I believe that was an error from merging down: seems like `summary_state_view_old` should also be removed.

Reply:
Confirmed Daniel fixed this.